pallets/staking/src/lib.rs

```diff
@@ -1037,6 +1038,21 @@ pub mod pallet {
             Self::do_staking_delegate(who, dst_level)
         }
 
+        #[pallet::weight(T::WeightInfo::usdt_staking_delegate())]
+        #[transactional]
+        pub fn usdt_staking_delegate(
+            origin: OriginFor<T>,
+            account_id: T::AccountId,
+            usdt_amount: BalanceOf<T>,
```

Comment:
`USDT` only has 6 decimals, see https://etherscan.io/token/0xdac17f958d2ee523a2206206994597c13d831ec7
how about using a `u32` or `usize` type.

Reply:
according to our smart contract

Reply:
not a big deal, keep BalanceOf<T> is ok for me.